force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -121,7 +139,8 @@ public with sharing class VariablesDataTypesOperators {
 	 * @return The complete sentence as a single String.
 	 */
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
+        return 'The' + ' ' + noun + ' ' + 'is' + ' ' + verb + '' + endingPunctuation;
```

Comment:
See if you can add the spaces to the variables and remove some of the extra + signs